### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by DJMogens**
@Tinggaard you may review if ya want

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,67 @@
...
+    def get(self):
+        return {'message': 'This is a new endpoint!'}
+```
+Remember to add any necessary input validation, error handling, and documentation.
+
+
+### Running the application
+
+```bash
+flask --app .src/main.py run
```
Comment (Tinggaard) [inline README.md:52]: The setup script is not working as intended if you need to specify the app here.

File: requirements.txt
Comment (Tinggaard) [inline requirements.txt:1]: The amount of dependencies is enormous. Only specify the ones you installed to get it to work (likely `flask`, `azure`, and a few others).
See also how it's done in the frontend for inspiration on how to work out the dotenv easily https://github.com/cvpl-fdca/SagaLabs-frontend

File: setup.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+virtualenv venv
+source venv/bin/activate
```
Comment (Tinggaard) [inline setup.sh:4]: This only sources the file for when we install below. It needs to be sourced again after running the script.

File: requirements.txt
Comment (DJMogens) [reply requirements.txt:1]: I simply used pip freeze > requirements. Alle the requirements are things that comes from dependencies of the others. but meaby we should use pipreqs instead: https://towardsdatascience.com/goodbye-pip-freeze-welcome-pipreqs-258d2e7a5a62

File: setup.sh (hunk 2 above)
Comment (DJMogens) [reply setup.sh:4]: help fix. i dont know a lot about the venv's in python

File: README.md (hunk 1 above)
Comment (DJMogens) [reply README.md:52]: That is true. The setup doesnt set the environment variable. $env:FLASK_APP = "main.py". Does it work on linux?

**Review by Tinggaard — APPROVED** [review]
I cleaned up a bit, let me know if it creates problems

**Comment by DJMogens** [comment]
> I cleaned up a bit, let me know if it creates problems.
Perfect. Tested good from my side. LGTM

**Pull request merged**